recipe/wrapper.c

```diff
@@ -0,0 +1,261 @@
+/*
+    Wrapper for Windows to avoid using bat files
+
+    To build/rebuild with mingw32, do this in the setuptools project directory:
+
+       gcc -DGUI=0           -mno-cygwin -O -s -o setuptools/cli.exe launcher.c
+       gcc -DGUI=1 -mwindows -mno-cygwin -O -s -o setuptools/gui.exe launcher.c
+*/
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <windows.h>
+#include <tchar.h>
+#include <fcntl.h>
+#include <process.h>
+
+// http://www.catch22.net/tuts/reducing-executable-size
+// Favour small code
+#pragma optimize("gsy", on)
```

Comment:
I had a lot more here, but it basically all errored when I compiled it with one of the older versions of VS...

Currently it's 124kb (VS) vs 25kb (gcc), but gcc needs the R channel :-( 

@mingwandroid Any chance of seeing the m2 packages in conda-forge?